test/hotspot/jtreg/vmTestbase/nsk/monitoring/share/thread/ThreadUtils.java

```diff
@@ -218,20 +228,33 @@ public static void threadDump(Log log, ThreadInfo[] tinfos) {
                 }
         }
 
+        // Most uses of waitThreadState usually succeed without retries.
+        // Sleep time increased to help avoid spurious failures.
```

Comment:
"increased" doesn't mean anything outside this PR as you can't tell the value has changed.

Reply:
Also any comment on sleep time belongs after line 233.